gen-flow.sh

```diff
@@ -1,4 +1,4 @@
 sed '1,2d' src/term-spec.js > dist/term-spec.js.flow;
-echo "declare module 'sweet-spec' {\n" | cat - dist/term-spec.js.flow > out.js;
-echo "\n}" >> out.js;
+echo "// @flow\n" | cat - dist/term-spec.js.flow > out.js;
+echo "\n" >> out.js;
```

Comment:
without the `// @flow` flow never runs on the file. The declare module is only necessary if you're putting this file in a `flow-typed` directory, otherwise normal node module resolution gives the module name just fine.